### ISSUE TITLE
Use global price data instead of local

### ISSUE THREAD

**Issue by ExPrgrmmr** [bisq-network/bisq#3527]
**Bitcoin trading in minor currencies** got low total volume on exchanges which makes it unsutable to have the local price as reference on a trading platform.

**The consequence is that the local price** is not properly aligned to market changes and appears way off. Note that this is not the case for major currencies with high total volume where local price approximates the global price 

**So instead of using local price** the global price can be used in Bisq to give the best possible trading experience for everybody.

**Only a small change** in the price-node is needed to achieve this. 

**Forum discussion**
https://bisq.community/t/can-we-use-bitcoinaverage-global-price/7474

**Comment by ExPrgrmmr** [comment]
We don't use Bisq's local prices (other than for BSQ). Now step up from BitcoinAverage local to global. How is one supposed work as a "Market  Maker"  with this flaw? https://github.com/bisq-network/growth

**Comment by chimp1984** [comment]
@BisqNoMore 
> We don't use Bisq's local prices

You confuse Bitcoin Average Local price with Bisq's price.

**Comment by Bisq2People** [comment]
No, but agree it was a bad choosen example to compare Bisq with BitcoinAverage local.
